superchain/configs/mainnet/redstone.toml

```diff
@@ -13,6 +13,11 @@ seq_window_size = 3600
 max_sequencer_drift = 600
 data_availability_type = "eth-da"
```

Comment:
Thanks for highlighting.

I think this is because of the renaming breaking change -- did you migrate your `rollup.json` yet?
https://docs.optimism.io/builders/chain-operators/features/alt-da-mode#modify-rollupjson-config

I have filed a new issue https://github.com/ethereum-optimism/superchain-registry/issues/557 so we can do a better job communicating this.